internal/ent/entc.go

```diff
@@ -45,16 +44,34 @@ func main() {
 	// Ensure the schema directory exists before running entc.
 	_ = os.Mkdir("schema", 0755)
 
-	// Add OpenAPI Gen extension
-	spec := new(ogen.Spec)
-	oas, err := entoas.NewExtension(
-		entoas.Spec(spec),
+	ex, err := entoas.NewExtension(
+		entoas.SimpleModels(),
 		entoas.Mutations(func(graph *gen.Graph, spec *ogen.Spec) error {
-			spec.Info.SetTitle("Datum API").
+			spec.SetOpenAPI("3.1.0")
+			spec.SetServers([]ogen.Server{
+				{
+					URL:         "https://api.datum.net",
+					Description: "Datum Production API Endpoint",
+				},
+				{
+					URL:         "http://localhost:17608/v1",
```

Comment:
Without the v1 to align with the production server?

```suggestion
					URL:         "http://localhost:17608,
```

Reply:
prod server should actually have the v1, not the other way around